### ISSUE TITLE
Vip God mode working while pvping

### ISSUE THREAD

**Issue by staii7** [Linkupdated/LaCedule#92]
This is OP because one could take no fall damage plus be a pvp race such as orc or vampire.

**Comment by Linkupdated** [comment]
people can't while in god... ?

**Comment by Linkupdated** [comment]
![God Work as intended](http://i.imgur.com/zD5qkJg.png)

**Comment by chaddy12** [comment]
![rr](https://cloud.githubusercontent.com/assets/11859130/7339072/b8355206-ec1e-11e4-82db-84d02eed3d8e.PNG)
What the wiki says vips take no fall damage while in god, also they can pvp while in god

**Comment by Linkupdated** [comment]
as do hawkman tho

**Comment by chaddy12** [comment]
yeah but hes saying as you are a vip you can have god mode on and not take fall damage and pvp also can be another race. While bvips cant attack in god, so while they are out of god they take fall damage, Maybe you can add on to vip the no attacking in god mode? That will solve the issue without removing fall damage from vip god

**Comment by Linkupdated** [comment]
thanks ill fix that soon
